### PR TITLE
#7 Make delete swift, callback to flutter

### PR DESCRIPTION
**Pull request by ivofernandes**
I'm using this package in a project project from my company, and I need to update the cloudkit delete function so I receive a response callback when the icloud responds, hope you enjoy

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 1.1.1
+Make delete and delete all swift code callback to flutter
```
Comment (Tutorialwork) [inline CHANGELOG.md]: Please make sure the formatting is the same as in 1.1.0

File: ios/Classes/SwiftCloudKitPlugin.swift [hunk 2]
```diff
@@ -45,93 +45,97 @@ public class SwiftCloudKitPlugin: NSObject, FlutterPlugin {
...
                     }
-                    
-                    
                 })
-                
-                
+
+                deleteGroup.notify(queue: .main) {
+                    if deletedRecord {
+                        result(true)
```
Comment (Tutorialwork) [inline ios/Classes/SwiftCloudKitPlugin.swift:99]: Are you returning now an boolean if the deletion was successfully? Then you also need to adapt the return type of the delete function in the cloud_kit.dart file.

Comment (ivofernandes) [reply ios/Classes/SwiftCloudKitPlugin.swift:99]: Good idea

File: CHANGELOG.md (hunk 1 above)
Comment (ivofernandes) [reply CHANGELOG.md]: Do you mean to look like this:

## 1.1.1
* Make delete and delete all swift code callback to flutter

## 1.1.0

* Add null safety support
* Add comments to functions

## 1.0.0

* The first release of the cloud_kit plugin.

**Comment by ivofernandes** [comment]
Okay now it's passing the boolean to the flutter response :)

**Comment by Tutorialwork** [comment]
Great, I'll merge it. 👍🏻

**Pull request merged**